.github/workflows/rust.yml

```diff
@@ -53,4 +61,3 @@ jobs:
           toolchain: ${{ matrix.rust }}
 
       - name: Cross compile
-        run: cargo test --verbose --features ${{ matrix.feature }}
```

Comment:
Question: Why did you remove the cross compilation tests?

Reply:
hmm my bad, I didn't see that. Will bring it back :-)

Reply:
@Davidson-Souza done!